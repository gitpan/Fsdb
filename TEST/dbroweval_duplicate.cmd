prog='dbroweval'
args='-f TEST/dbroweval_duplicate.code'
cmp='diff -cb '
in=TEST/dbroweval_ex.in
