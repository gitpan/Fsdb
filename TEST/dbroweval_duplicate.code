&$write_fastpath_sub($fref);
