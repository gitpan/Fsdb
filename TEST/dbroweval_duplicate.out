#fsdb size mean stddev pct_rsd
1024	1.4962e+06	2.8497e+05	19.047
1024	1.4962e+06	2.8497e+05	19.047
10240	5.0286e+06	6.0103e+05	11.952
10240	5.0286e+06	6.0103e+05	11.952
102400	4.9216e+06	3.0939e+05	6.2863
102400	4.9216e+06	3.0939e+05	6.2863
#  | dbsetheader size bw
#  | /home/johnh/BIN/DB/dbmultistats size bw
#  | /home/johnh/BIN/DB/dbcol size mean stddev pct_rsd
#   | dbroweval   { &$write_fastpath_sub($fref);  }
